Doc/library/stdtypes.rst

```diff
@@ -3520,12 +3520,11 @@ copying.
    is a single byte, but other types such as :class:`array.array` may have
    bigger elements.
 
-   ``len(view)`` is equal to the length of :class:`~memoryview.tolist`.
-   If ``view.ndim = 0``, the length is 1. If ``view.ndim = 1``, the length
-   is equal to the number of elements in the view. For higher dimensions,
-   the length is equal to the length of the nested list representation of
-   the view. The :class:`~memoryview.itemsize` attribute will give you the
-   number of bytes in a single element.
+   ``len(view)`` is equal to the length of :class:`~memoryview.tolist`, which
+   is the nested list representation of the view. If ``view.ndim = 1``,
+   this is equal to the number of elements in the view.
```

Comment:
```suggestion
   this is equal to the number of elements in the view.
   
   .. versionchanged:: 3.9
       If ``view.ndim = 0``, ``len(view)`` now raises `TypeError` instead of returning 1.
```